ocs_ci/utility/utils.py

```diff
@@ -1516,44 +1516,51 @@ def get_next_version_available_for_upgrade(current_tag):
     raise TagNotFoundException("Couldn't find any tag!")
 
 
-def get_ocs_olm_operator_tags(limit=100):
+def load_auth_config():
     """
-    Query the OCS OLM Operator repo and retrieve a list of tags.
-
-    Args:
-        limit: the number of tags to limit the request to
+    Load the authentication config YAML from /data/auth.yaml
 
     Raises:
```

Comment:
This meant to be part of get_ocs_olm_operator_tags and not this load_auth_config  function I guess

Reply:
Yep, you're right - `FileNotFoundError` _should_ be here, the other two - shouldn't :)